drv/Map.drv

```diff
@@ -142,7 +210,459 @@ public interface MAP KEY_VALUE_GENERIC extends FUNCTION KEY_VALUE_GENERIC, Map<K
 
 	@Deprecated
 	@Override
-	boolean containsValue(Object value);
+	default boolean containsValue(Object ov) {
```

Comment:
Yes, for primitive values! But (as above) consider a `Int2ObjectMap`. Do you want to allow a `put(1, null)`? Then a `if (ov == null) return false` short-circuit is not correct.